09. cli-and-http-request/clihelper/CliBind.go

```diff
@@ -0,0 +1,17 @@
+package clihelper
+
+import (
+	"log"
+	"sync"
+)
+
+func CliBind(c CliBindingProperties, s stdInParameter) {
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go func() {
+		stdout, errStdout := AttachLoggers(c, s)
```

Comment:
let's move go and wg from here, and put `go` inline 12, only if Async is true, let's add one more parameter in the `Cmd Info` IsAsync

Reply:
move go to line 12? you mean like this ? >> stdout, errStdout := go AttachLoggers(c, s)